SeaPublicWebsite/Helpers/UserFlow/PageLinker.cs

```diff
@@ -0,0 +1,797 @@
+﻿using System;
+using Microsoft.AspNetCore.Routing;
+using SeaPublicWebsite.DataModels;
+using SeaPublicWebsite.Models.EnergyEfficiency.QuestionOptions;
+
+namespace SeaPublicWebsite.Helpers.UserFlow
+{
+    public interface IPageLinker
+    { 
+        public string BackLink(PageName page, UserDataModel userData, PageName? entryPoint = null, string from = null);
```

Comment:
I don't think we need `from` here. We store OwnershipStatus and Country in the UserDataModel so we should be able to tell where we came from. If OwnershipStatus is null and Country is filled in we must have come from country. If we reset ownership status when country changes in SEABETA-274 then this should be reliable.

Reply:
Yup, I can do that! (Another thing from the old code I missed)